### PR TITLE
bugfix:fix copy dir is "."

### PR DESCRIPTION
**Pull request by kakaZhou719**

### CODE REVIEW

File: build/instruction_handler.go [hunk 1]
```diff
@@ -173,10 +173,15 @@ func (h *handler) copyFiles(srcFileName, dstFileName, tempBuildDir string) error
 	}
 
 	if fi.IsDir() {
+		//default workdir is rootfs,so if copy dst is ".", name it as filepath.Base(srcFileName)
```
Comment (fanux) [inline build/instruction_handler.go:176]: If dst dir is not absolute path， using filepath.join($rootfs, dst), else using dst dir directly

Comment (kakaZhou719) [reply build/instruction_handler.go:176]: yes,we assume tempBuildDir is temp rootfs.so, the dst dir is filepath.join(tempBuildDir , dst).
if copy dir : 
1,copy . . :  src is context, dst is $rootfs
2,copy dir . : src is context/dir , dst is  $rootfs.
3,copy dirA dirB ： src is context/dirA , dst is  $rootfs/dirA/dirB

Comment (fanux) [reply build/instruction_handler.go:176]: COPY dir .       dst is $rootfs/dir is better I think. = cp -r dir ./
COPY dirA ./dirB dst is $rootfs/dirB/dirA is better. = cp -r dirA ./dirB
COPY dirA /dirB dst is /dirB/dirA = cp -r dirA /dirB

**Comment by kakaZhou719** [comment]
close it as we dont support system command now.

**Pull request closed without merging**